src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -1213,6 +1306,42 @@ input InputTriangle {
     timeFactor: Float
 }
 
+"""
+Relative importances of optimization factors. Only effective for bicycling legs.
+Invariant: `cyclingFriendliness + flatness + time == 1`
+"""
+input InputTriangleCyclingFactors {
+    """
+    Relative importance of cycling friendliness. Factors that are used to evaluate
+    cycling friendliness include safety, convenience and general cyclist preferences.
+    """
+    cyclingFriendliness: WeightingFactor
```

Comment:
Is friendliness fitting term for "safety"?

Reply:
we don't need "cycling"